HISTORY.md

```diff
@@ -1,5 +1,8 @@
 ### ensmallen ?.??.?: "???"
 ###### ????-??-??
+
+### ensmallen 2.15.0: "Why Can't I Manage To Grow Any Plants?"
```

Comment:
haha, you are not the only one.

Reply:
Yeah, so at least in my case, I think I kind of know the answer, which is that I mixed an entire 40-lb bag of fertilizer into the soil when it turns out the directions said I should only use 3 lbs of it.  It turns out reading is important...

Reply:
In my case, I sometimes forget to water or I water too much, I think either is not perfect.

Reply:
Now, seeds will be very happy with 40-lb of fertilizer.